src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -729,5 +733,25 @@ private static IEnumerable<string> GetClassesForType(Type type)
 				}
 			}
 		}
+
+		private InputCursor _protectedCursor = InputSystemCursor.Create(InputSystemCursorShape.Arrow);
+
+		protected InputCursor ProtectedCursor
```

Comment:
Does this add the API on UWP packages?

Reply:
Good catch, however as UI tests are still using the UWP namespace I would not bother with that.

Reply:
What about making it `private protected` on UWP and `protected` on WinUI. It would allow us to use it in UI tests and internally without having to add the `#if HAS_UNO_WINUI` everywhere.